manifests/params.pp

```diff
@@ -79,6 +88,10 @@
   $consul_template_version  = '0.14.0'
 
   $nginx_ensure             = 'installed'
```

Comment:
Maybe rename this it `nginx_package_ensure` now that we also have `nginx_service_ensure`?